api/kyverno/v2alpha1/global_context_entry_types.go

```diff
@@ -119,7 +119,7 @@ type GlobalContextEntryList struct {
 type KubernetesResource struct {
 	// Group defines the group of the resource.
 	// +kubebuilder:validation:Required
```

Comment:
LGTM! Can we just add `+optional` here and mark the validation `Optional`. I know it's not used but just for sematnics.